### PR TITLE
add navbar

### PR DESCRIPTION
**Pull request by Automatic365**

### CODE REVIEW

File: app/assets/css/fonts/HELP-US-OUT.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+I hope you love Font Awesome. If you've found it useful, please do me a favor and check out my latest project,
+Fort Awesome (https://fortawesome.com). It makes it easy to put the perfect icons on your website. Choose from our awesome,
+comprehensive icon sets or copy and paste your own.
+
+Please. Check it out.
+
+-Dave Gandy
```
Comment (dtinianow) [inline app/assets/css/fonts/HELP-US-OUT.txt]: What

File: app/assets/css/fonts/HELP-US-OUT.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+I hope you love Font Awesome. If you've found it useful, please do me a favor and check out my latest project,
```
Comment (Yoyo2Code) [inline app/assets/css/fonts/HELP-US-OUT.txt]: Might be awesome, but he must go!

File: app/assets/css/fonts/css/font-awesome.css [hunk 3]
```diff
@@ -0,0 +1,2199 @@
...
+}
+.sr-only-focusable:active,
+.sr-only-focusable:focus {
+  position: static;
+  width: auto;
+  height: auto;
+  margin: 0;
+  overflow: visible;
+  clip: auto;
+}
```
Comment (dtinianow) [inline app/assets/css/fonts/css/font-awesome.css]: Wow I am impressed you wrote all of this!

File: app/views/layouts/_navbar.html.erb [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+        <%= link_to "Login", login_path %>
+        <% end %>
+        <li><a href="/dashboard">Dashboard</a></li>
+        <li><a href="/orders">Order History</a></li>
+        <li role="separator" class="divider"></li>
+      </ul>
+    </li>
+  </ul>
+</div>
+</div>
```
Comment (dtinianow) [inline app/views/layouts/_navbar.html.erb]: Nice use of partials!

File: spec/models/role_spec.rb [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+RSpec.describe Role, type: :model do
+  context 'associations' do
+    it { should have_many(:business_roles) }
+    it { should have_many(:businesses).through(:business_roles) }
+  end
+
+  context 'validations' do
+    it { should validate_presence_of(:name) }
+  end
```
Comment (dtinianow) [inline spec/models/role_spec.rb]: This needs to go!

**Comment by dtinianow** [comment]
Role and businesses roles needs to eliminated

**Comment by dtinianow** [comment]
Really like use of the partial for the navbar

**Comment by Automatic365** [comment]
Can I get a fresh merge?

File: app/assets/stylesheets/custom.css.erb [hunk 6]
```diff
@@ -181,165 +141,19 @@ ol, ul {
...
-#houston-header {
-  background-image: url('houston-header.jpg');
-  background-position: center;
-  background-size: center;
-}
-#philly-header {
-  background-image: url('philly-header.jpg');
-  background-position: center;
-  background-size: center;
-}
```
Comment (dtinianow) [inline app/assets/stylesheets/custom.css.erb]: yeah good riddance

File: spec/features/visitor_can_see_all_locations_spec.rb [hunk 7]
```diff
@@ -2,21 +2,22 @@
...
-  scenario 'they can access individual locations' do
-    property = create(:property)
-
-    visit root_path
-    click_link property.location.city
-
-    expect(current_path).to eq("/locations/#{property.location.slug}")
-  end
+  # scenario 'they can access individual locations' do
+  #   property = create(:property)
```
Comment (dtinianow) [inline spec/features/visitor_can_see_all_locations_spec.rb]: This is nonsense Yoseph

**Comment by dtinianow** [comment]
Fresh merges for all!

**Pull request merged**